docs/pax-table-format.md

```diff
@@ -0,0 +1,487 @@
+| `pax_bloom_filter_work_memory_bytes` | `int`      | Range `[1024, 2147483647]` (1KiB to 2GiB). The default value is `10240` (10KiB).         | Controls the maximum memory allowed for bloom filter usage.                                                                                                                                                                     |
+
+## Best practices
+
+- Use partitioning options:
+
+    - It is recommended to use partitioning options when data needs to be imported based on an integer column and meets the following conditions:
+
+        - The data is evenly distributed across this column, with a wide range and no extreme concentration.
+        - This column is often used as a filter condition in queries or as a key for joins.
+
+    - Keep in mind that the PAX partition key is only effective during a single batch data import. It cannot be adjusted between multiple data writes. The partition key settings only apply to future inserts or updates, so after changing the partition key, newly imported data will follow the new partition key.
+
+- Use `minmax` statistics:
+
+    - For columns with a wide data range that are often used in query filters, setting `minmax` values for these columns can greatly speed up the query process.
+    - By using `minmax` statistics, if a column in a data file does not match the `minmax` values or null tests, the entire file can be skipped quickly, avoiding unnecessary data scans.
+    - Important note: The effectiveness of `minmax` depends on how data is inserted. If the data in a PAX table is inserted in batches (such as with `batch insert` or `copy`) and the data range within each batch is continuous, then `minmax` will be very effective. However, if the data is inserted randomly, the effectiveness of `minmax` filtering may be reduced.
+
+- If a PAX table has multiple columns used in query filters, you can use `with(cluster_columns='b,c,d', cluster_type='zorder')`. After sorting the data with `zorder` encoding, any of the `cluster_columns` can benefit from filtering. In comparison, for single-column-based cluster sorting, filtering is only effective on the sorted key, with minimal impact on other columns. Clustering sorting influences `minmax`-based filtering but does not affect `bloom filter`-based filtering.
+
+- If your query uses filters like `column1 in (12, 11, 13)` or `column1 = 'X'`, you might consider using a `bloom filter (with(bloomfilter_columns='b,c'))`. In internal implementation, PAX calculates bloom filters for certain data. If the bloom filter can confirm that the data is not in the block, PAX can skip the current block. Note that the effectiveness of bloom filter depends on several factors:
+
+    - The amount of data within a `group`.
+    - The space used by the `bloom filter`.
```

Comment:
Also, there are some dev docs on PAX in the main repo. Will we add them to the `Developer` module? If not, can add their links here for users and developers' references. FYI.